### PR TITLE
glsl-in: Add support for texelFetchOffset

### PR DESCRIPTION
**Pull request by JCapucho**

### CODE REVIEW

File: src/front/glsl/builtins.rs [hunk 1]
```diff
@@ -322,7 +323,7 @@ pub fn inject_builtin(
                 let dim_value = image_dims_to_coords_size(dim);
                 let coordinates = make_coords_arg(dim_value + arrayed as usize, Sk::Sint);
 
-                let args = vec![
+                let mut args = vec![
```
Comment (kvark) [inline src/front/glsl/builtins.rs:326]: perhaps we should do `ArrayVec` instead?

**Review by kvark — COMMENTED** [review]
Are these clippy warnings new?

**Comment by kvark** [comment]
We should probably run Clippy on MSRV only, so that it's predictable.

File: src/front/glsl/builtins.rs (hunk 1 above)
Comment (JCapucho) [reply src/front/glsl/builtins.rs:326]: I would fine with doing this in another PR but I don't think mixing unrelated changes in the same PR would be a good idea.

**Comment by JCapucho** [comment]
@kvark sorry for the late response, was out of town for a while, jim seems to have already fixed the clippy lint but it might still be a good idea to change the pipeline to use the MSRV or the latest stable (and maybe have another optional pipeline for checking on nightly), I can implement it if you want, just wanted to hear your thoughts on the matter.

**Pull request merged**